apps/web/app/(navigation)/SideNavigationBar.tsx

```diff
@@ -237,6 +237,7 @@ export default function SideNav() {
         isNavListItemClicked={isNavListItemClicked}
         resetIsNavListItemClicked={resetIsNavListItemClicked}
         refetchBudgetsFunction={refetch}
+        resetSearch={setSearchValue}
```

Comment:
You are calling a function without a parameter, which causes setting this state to `undefined`, while it should be of a `string` type. Doesn't seem to break anything, but let's be on the safe side and have the proper type 🤓 

```suggestion
        resetSearch={() => searchValue("")}
```

<img width="947" alt="search1" src="https://github.com/intive/patronage2023-js/assets/80450315/e548860d-8e57-4bd2-808b-8a2204129471">

after closing the side nav, the state is `undefined` ⬇️ 

<img width="957" alt="search2" src="https://github.com/intive/patronage2023-js/assets/80450315/0ce13db7-7621-42ed-8bac-2b7434ebd4ad">

Reply:
Great idea. Fixed 🙂